src/main/java/io/shiftleft/controller/AccountController.java

```diff
@@ -39,8 +54,29 @@ public Account createAccount(Account account) {
     }
 
     @GetMapping("/account/{accountId}")
-    public Account getAccount(@PathVariable long accountId) {
+    public Account getAccount(@PathVariable long accountId) throws SQLException {
         log.info("Account Data is {}", this.accountRepository.findOne(1l).toString());
+
+        try {    
+            getConnection();
+
+            String sql = "SELECT * FROM ACCOUNT WHERE ACCOUNTID = '" + accountId;
+            preparedStatement = connection.prepareStatement(sql);

```

Comment:
Add a comment here explaining prepareStatement is ineffective since the concatenation has already happened?

Reply:
OK, doing it